File: 5th Semester/Programming Paradigms/NP2 Exercises - Part 2/ex14.py
# Escreva um programa que leia números inteiros do teclado. O programa deve ler os números até que
# o usuário digite 0 (zero). No final da execução, exiba a quantidade de números digitados, assim como a soma
# e a média aritmética. 

num = 1
quantidade = 0
total = 0
while (num != 0):
    num = int(input('(0 para parar)\nInsira numeros: '))
    total += num
    quantidade += 1

print(f'Você inseriu {quantidade} números e estes somam {total}')

File: 5th Semester/Programming Paradigms/NP2 Exercises - Part 2/ex14.c
// Escreva um programa que leia números inteiros do teclado. O programa deve ler os números até que
// o usuário digite 0 (zero). No final da execução, exiba a quantidade de números digitados, assim como a soma
// e a média aritmética.

#include <stdio.h>

int main()
{
    int num, quantidade = 0, total = 0;
    do
    {
        printf("(0 para parar)\nInsira numeros: ");
        scanf("%i", &num);
        total += num;
        quantidade++;
    } while (num != 0);
    printf("Você inseriu %i números e estes somam %i", quantidade, total);
    return 0;
}